src/steps/Microsoft.DotNet.UpgradeAssistant.Steps.Packages/Analyzers/TargetCompatibilityReferenceAnalyzer.cs

```diff
@@ -51,7 +51,7 @@ public async Task AnalyzeAsync(IProject project, IDependencyAnalysisState state,
                 else
                 {
                     // If the package won't work on the target Framework, check newer versions of the package
-                    var newerVersions = await _packageLoader.GetNewerVersionsAsync(packageReference, state.TargetFrameworks, true, token).ConfigureAwait(false);
+                    var newerVersions = await _packageLoader.GetNewerVersionsAsync(packageReference, state.TargetFrameworks, new() { Prerelease = true }, token).ConfigureAwait(false);
```

Comment:
I think the `true` that was passed here previously corresponded to `latestMinorAndBuildOnly`, not `prerelease`.

Reply:
Mike is right, it was for latestMinorAndBuildOnly